bin/cron/workshops_to_s3

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+require_relative '../../dashboard/config/environment'
+require_relative '../../lib/cdo/only_one'
+require 'cdo/aws/s3'
+
+# Automates the export of 5-day summer workshops data
+# to an S3 bucket. Mirrors what would happen via the UI
+# if an admin clicked the "Download CSV" option
+# in the workshop dashboard.
+def main
+  workshops = Pd::Workshop.
+    where(
+      {
+        course: [Pd::Workshop::COURSE_CSP, Pd::Workshop::COURSE_CSD],
+        subject: Pd::Workshop::SUBJECT_SUMMER_WORKSHOP
+      }
+    ).
+    scheduled_start_on_or_after(Date.new(2019, 5, 1))
+
+  workshop_hash = workshops.map {|w| Api::V1::Pd::WorkshopDownloadSerializer.new(w).attributes}
+
+  # Generate throwaway class that has functionality
+  # from Api::CsvDownload module to emulate what happens in
+  # controller/api/vi/pd/workshop_controller's filter method
+  output = Class.new.extend(Api::CsvDownload).generate_csv workshop_hash
```

Comment:
I think you should be able to call `include Api::CsvDownload` inside of `main` and then use `generate_csv` [(example)](https://github.com/code-dot-org/code-dot-org/blob/staging/bin/cron/calculate_workshop_survey_results)